src/mono/mono/component/debugger-agent.c

```diff
@@ -8907,7 +8907,7 @@ thread_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 		if (!s) {
 			buffer_add_int (buf, 0);
 		} else {
-			const size_t len = strlen (s);
+			const guint32 len = (guint32)strlen (s);
```

Comment:
Did you intend to have pattern doing type cast on the line used?

Reply:
Yup.